### PR TITLE
fix spell error

### PR DESCRIPTION
**Pull request by eacdy**
fix spell error

### CODE REVIEW

File: docs/src/main/asciidoc/spring-cloud-netflix.adoc [hunk 1]
```diff
@@ -1549,9 +1549,10 @@ Example configuration:
 In this example we are strangling the "legacy" app which is mapped to
 all requests that do not match one of the other patterns. Paths in
 `/first/{all}` have been extracted into a new service with an external
-URL. And paths in `/second/{all}` are forwared so they can be handled
+URL. And paths in `/second/{all}` are forwarded so they can be handled
 locally, e.g. with a normal Spring `@RequestMapping`. Paths in
-`/third/{all}` are also forwarded, but with a different prefix
+`/third/{all}` are also
```
Comment (ryanjbaxter) [inline docs/src/main/asciidoc/spring-cloud-netflix.adoc]: why this change?

**Comment by ryanjbaxter** [comment]
@eacdy bump ^^^

**Comment by spencergibb** [comment]
merged with polish via 6fc5d1aea39241f178f7172492dee83b07fa67f0
